### PR TITLE
Login consent screen

### LINKED ISSUE #1424: Government Consent Screen before User Login
Government Consent Screen should appear prior to user logging into the app (ATO Requirement).

Current recommendation is to take the one from cloud.gov, but need to be have CMS lawyers approval. Also available, is the language from NIST. 

Reference: https://login.fr.cloud.gov/login

### This task is done when...
 - [ ] User is presented with the Government Consent Screen
 - [ ] User can access the login screen after the Consent Screen

### PR DESCRIPTION
**Pull request by mgwalker**
### This pull request changes...
- adds a government IS consent banner before allowing access to the web app
  - Once a user logs in, we store a cookie in their browser that's valid for 3 days and won't show them the consent banner again as long as the cookie is valid.  The cookie expiration is extended when the user logs in again, but not every time they visit the app.
- closes #1424

### This pull request is ready to merge when...
- [ ] Tests have been updated (and all tests are passing)
- [x] This code has been reviewed by someone other than the original author
- [ ] The experience passes a basic manual accessibility audit (keyboard nav, screenreader, text scaling) OR an exemption is documented

### This feature is done when...
- [ ] Design has approved the experience
- [ ] Product has approved the experience

### CODE REVIEW

File: web/src/components/ConsentBanner.js [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+      this.agreeAndContinue();
+      return null;
+    }
+
+    return (
+      <div className="card--container">
+        <div className="ds-l-container">
+          <div className="ds-l-row card">
+            <div className="ds-l-col--1 ds-u-margin-left--auto" />
+            <div className="ds-l-col--12 ds-l-sm-col--10 ds-l-lg-col--6">
```
Comment (mgwalker) [inline web/src/components/ConsentBanner.js]: This is all borrowed from the admin form card work @hbillings did.

**Comment by mgwalker** [comment]
Added the header and tweaked the agree button text.

@jeromeleecms Does the "CMS HITECH APD" header work?  It uses the same header that the admin forms do, so hopefully as the site-wide header is developed and refined, this will get those benefits (but without any links to anything, since you haven't consented yet...!)

**Comment by jeromeleecms** [comment]
The header "CMS HITECH APD" works. We have some branding questions down the road that can better inform this. 

Can you update the color of the header to the one that Austin put in for #1468 ("primary darkest")? That way we won't have to come back and do it later.

**Comment by mgwalker** [comment]
It's the same header, so when #1468 is implemented, this one should be updated for free.  😉

File: web/src/components/ConsentBanner.js (hunk 1 above)
Comment (hbillings) [reply web/src/components/ConsentBanner.js]: aww yiss!!

File: web/src/components/ConsentBanner.js [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+              {showDetails ? (
+                <Fragment>
+                  <p>
+                    This warning banner provides privacy and security notices
+                    consistent with applicable federal laws, directives, and
+                    other federal guidance for accessing this Government system,
```
Comment (hbillings) [inline web/src/components/ConsentBanner.js]: @jameshupp do we need to caps `Government` in any of the instances here?

Comment (mgwalker) [reply web/src/components/ConsentBanner.js]: If yes, let's create an issue to correct it!

**Pull request merged**